File: server-py/src/model/TrackEntity.py
from .. import db


class TrackEntity(db.Model):
    __tablename__ = "track"

    id = db.Column(db.Integer, primary_key=True)
    counter = db.Column(db.Float, nullable=False)
    createdat = db.Column(db.DateTime, nullable=False)

    def __repr__(self):
        return f'Track id:{self.id}, counter:{self.counter}, createdat:{self.createdat}'

    def to_dict(self):
        return {
            'id': self.id,
            'counter': self.counter,
            'createdat': self.createdat
        }

File: server/src/main/java/com/ckinan/cliff/TrackEntity.java
package com.ckinan.cliff;

import javax.persistence.*;
import java.util.Date;

@Entity
@Table(name = "track")
public class TrackEntity {

    @Id
    @GeneratedValue(strategy= GenerationType.IDENTITY)
    private Long id;
    @Column(nullable = false, unique = true)
    private Double counter;
    @Column(name = "createdat")
    private Date createdAt;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public Double getCounter() {
        return counter;
    }

    public void setCounter(Double counter) {
        this.counter = counter;
    }

    public Date getCreatedAt() {
        return createdAt;
    }

    public void setCreatedAt(Date createdAt) {
        this.createdAt = createdAt;
    }
}